include/arc_utilities/ros_helpers.hpp

```diff
@@ -115,6 +132,40 @@ namespace ROSHelpers
             return Maybe::Maybe<T>();
         }
     }
+
+    template <typename T>
+    inline Maybe::Maybe<std::vector<T>> GetVectorOptional(const ros::NodeHandle& nh, const std::string& param_name, const std::string& calling_fn_name)
+    {
+        ROS_DEBUG_STREAM_NAMED("params", "No default value for " << param_name << ": Value must be on paramter sever");
+        std::vector<T> vec;
+        if (nh.getParam(param_name, vec))
+        {
+            ROS_INFO_STREAM_NAMED("params", "Retrieving " << std::left << std::setw(PARAM_NAME_WIDTH) << param_name << " of length " << vec.size());
```

Comment:
`ROS_DEBUG_...`

Reply:
I like having a log of what params were looked up as part of the program output, without enabling DEBUG logging. I'm going to leave this one as `ROS_INFO...` unless you have a strong objection.

Reply:
I'd rather debug, but I don't use these much so your preference should take precedence here

However, things should be consistent. Why does line 97 use `DEBUG`?

Reply:
Nothing to see here, move along...